website/static/downloads/swagger.yaml

```diff
@@ -1110,7 +1110,22 @@ paths:
           description: resource not found
         "409":
           description: invalid argument
-   
+  '/exchange/mosaics':
+    get:
+      tags:
+      - Exchange routes
+      summary: Get a list of mosaics id with exchange offers
+      description: Get a list of mosaics id with exchange offer 
+      operationId: getExchangeMosaics
+      responses:
+        "200":
+          description: success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/MosaicDTO"  
```

Comment:
as I know it's not MosaicDTO it's just a list of mosaics IDs. Instead of `$ref: "#/components/schemas/MosaicDTO"` should be:
```
type: object
properties:
  mosaicId:
    type: string
    example: "0DE5F42A0A3B4200"
```